requests_toolbelt/auth/guess.py

```diff
@@ -77,3 +77,72 @@ def __call__(self, request):
 
         request.register_hook('response', self.handle_401)
         return request
+
+
+class GuessProxyAuth(GuessAuth):
+    """
+    Guesses the auth type by WWW-Authentication and Proxy-Authentication
+    headers
+    """
+    def __init__(self, username=None, password=None,
+                 proxy_username=None, proxy_password=None):
+        GuessAuth.__init__(self, username, password)
```

Comment:
Why not use super here? The library doesn't support python 2.5 and as far as I remember, `AuthBase` from requests inherits from `object` so there should be no worry about being able to use super in this case.